mlir/test/Conversion/TosaToLinalg/tosa-to-linalg.mlir

```diff
@@ -545,6 +545,14 @@ func.func @test_simple_f32(%arg0: tensor<1xf32>) -> () {
   // CHECK: math.erf
   %24 = tosa.erf %0 : (tensor<1xf32>) -> tensor<1xf32>
 
+  // CHECK: linalg.generic
+  // CHECK: sin
```

Comment:
nit: the other checks in this file prefix the op with its dialect sin -> math.sin, likewise for cos.